test/repl.jl

```diff
@@ -493,11 +498,12 @@ for prompt = ["TestΠ", () -> randstring(rand(1:10))]
 
     # Try entering search mode while in custom repl mode
     LineEdit.enter_search(s, custom_histp, true)
+    end
 end
 
 # Test removal of prompt in bracket pasting
 begin
-    stdin_write, stdout_read, stderr_read, repl = fake_repl()
+    fake_repl() do stdin_write, stdout_read, repl
```

Comment:
I could, but I'm either going to change all of them or none of them. There's no sense in doing half of them.